data/products/sap/sle15/packages.yaml

```diff
@@ -0,0 +1,47 @@
+packages:
+  bootstrap:
+    SAP:
+      - sle-ha-release
+      - sle-module-live-patching-release
+      - sle-module-sap-applications-release
```

Comment:
duplicates `data/products/sap/ondemand/sle15/packages.yaml`, I am not certain I understand the slicing between BYOS and on-demand in this respect.

Reply:
Yes, this should only be in ondemand, my bad. Wrt slicing, I only replicated what's in the current image descriptions. The release packages of the various modules are included in the on-demand version but not in the BYOS ones. It's probably a good idea to not include release packages in BYOS images, as they should only be installed when the according module is enabled. Not sure whether including them in on-demand images is necessary. I would expect them to be installed automatically when the modules get enabled during registration.

Reply:
The registration is driven of the included product packages which is why the release packages are in the image builds for on-demand. The classification as provided by SCC/RMT as to product <-> module relationship and what may and may not be enabled by default is exceedingly complicated and may lead to a "not mirrored" issue.